packages/inter-protocol/test/psm/test-psm.js

```diff
@@ -147,186 +140,289 @@ test.before(async t => {
   t.context = await makeTestContext();
 });
 
-test('simple trades', async t => {
+/**
+ *
+ * @param {import('ava').ExecutionContext<Awaited<ReturnType<makeTestContext>>>} t
+ * @param {{}} [customTerms]
+ */
+async function makePsmDriver(t, customTerms) {
   const {
     zoe,
-    privateArgs,
+    feeMintAccess,
+    initialPoserInvitation,
     terms,
     installs: { psmInstall },
-    run,
-    anchorKit: { brand: anchorBrand, issuer: anchorIssuer, mint: anchorMint },
+    anchor,
   } = t.context;
-  const { publicFacet, creatorFacet } = await E(zoe).startInstance(
+
+  const mockChainStorage = makeMockChainStorageRoot();
+
+  /** @type {Awaited<ReturnType<import('../../src/psm/psm.js').start>>} */
+  const { creatorFacet, publicFacet } = await E(zoe).startInstance(
     psmInstall,
-    harden({ AUSD: anchorIssuer }),
-    terms,
-    privateArgs,
-  );
-  const giveAnchor = AmountMath.make(anchorBrand, 200n * 1_000_000n);
-  const seat1 = await E(zoe).offer(
-    E(publicFacet).makeSwapInvitation(),
-    harden({ give: { In: giveAnchor } }),
-    harden({ In: anchorMint.mintPayment(giveAnchor) }),
+    harden({ AUSD: anchor.issuer }),
+    { ...terms, ...customTerms },
+    harden({
+      feeMintAccess,
+      initialPoserInvitation,
+      storageNode: mockChainStorage.makeChildNode('thisPsm'),
```

Comment:
in `startPSM` the node path is `psm.<stable>.<anchor>`. There was no no unit test of `startPSM` and this PR doesn't add one. So the test plan includes manually testing on-chain, which is fuller coverage.